src/OpenTelemetry/OpenTelemetry.csproj

```diff
@@ -4,6 +4,7 @@
     <TargetFrameworks>$(TargetFrameworksForLibrariesExtended)</TargetFrameworks>
     <Description>OpenTelemetry .NET SDK</Description>
     <MinVerTagPrefix>core-</MinVerTagPrefix>
+    <IsPackable>true</IsPackable>
```

Comment:
@RichardChukwu Please stop pushing changes to this file. Or any other files 🤣 All of this is going to be handled via the props. I'm working to validate it works when rendered on NuGet.

Reply:
ooh, I'm sorry about that, didn't know @CodeBlanch